### PR TITLE
HAAR-1402: Scheduling downtime when database is also scheduled to be switched off

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -1,8 +1,13 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 generic-service:
+
```
Comment (karenmillermoj) [inline helm_deploy/values-preprod.yaml:2]: Should this yaml file also be defining the replicaCount ?
Obv this is nothing to do with your change

**Review by karenmillermoj — APPROVED** [review]
lgtm

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (brightonsbox) [reply helm_deploy/values-preprod.yaml:2]: As discussed, default in generic chart is 4 but we have agreed that for a service getting so little use, 2 should be fine.

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**